web/beacon-app/src/features/auth/components/Register/RegistrationForm.tsx

```diff
@@ -84,7 +93,7 @@ function RegistrationForm({ onSubmit }: RegistrationFormProps) {
             fullWidth
             data-testid="email"
             errorMessage={touched.email && errors.email}
-            errorMessageClassName="py-1"
+            errorMessageClassName="py-2"
```

Comment:
I think this kind of modification should not be specific to a single component

Reply:
You are right thanks for catching that. I add this directly to the design system package.